### PR TITLE
exploring: Mention Fedora packages repository

### PR DESCRIPTION
**Pull request by philn**
.

----

Site preview: https://igalia.github.io/wpewebkit.org/philn-patch-2/

### CODE REVIEW

File: about/exploring.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ All of this said, we believe that enabling developers to explore the space is im
   * [Ubuntu](https://packages.ubuntu.com/search?keywords=wpewebkit&searchon=sourcenames&suite=all&section=all)
   * [Raspbian](https://archive.raspbian.org/raspbian/pool/main/w/wpewebkit/)
   * [Arch Linux](https://archlinux.org/packages/extra/x86_64/wpewebkit/)
+  * [Fedoar](https://copr.fedorainfracloud.org/coprs/philn/wpewebkit/)
```
Comment (aperezdc) [inline about/exploring.md:48]: Typo: `Fedoar` → `Fedora`.

**Review by aperezdc — APPROVED** [review]
Please fix the typo before merging, and thanks for the patch!

**Comment by philn** [comment]
CI looks broken :(

**Comment by aperezdc** [comment]
> CI looks broken :(

Fixed in https://github.com/Igalia/wpewebkit.org/pull/108 :)

**Pull request merged**